debian/clearwater-etcd.init.d

```diff
@@ -231,6 +260,23 @@ do_start()
                 || return 2
 
         wait_for_etcd
+
+        # See that etcd stays up for more than a few seconds. There have been times when
+	# we get into an infinite loop restarting etcd when its configuration
+	# won't allow it to come up. The following looks for a specific error
+	# at the end of etcd's log file and if it's there, we'll take action and
+	# prevent the loop from continuing.
+        for i in {1..5}; do
+          if ! nc -z $listen_ip 4000; then
+	    if tail -1 /var/log/clearwater-etcd/clearwater-etcd.log | grep -q "etcdserver: the data-dir used by this member must be removed[.]"; then
+		echo "etcd didn't stay up due to data-dir error - removing it..."
+		rm -rf $DATA_DIR/$advertisement_ip
```

Comment:
I think you should delete all of `$DATA_DIR`, I don't think the internal structure of that folder is specified.